pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[tool.poetry]
+name = "up42-py"
+version = "0.32.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12"
+requests =  "2.31.0"
```

Comment:
is there a reason we want to pin these dependencies to this specific version? If not, let's make it flexible by `"^2.31.0"`

Same for rest of the dependencies.